### PR TITLE
Document the ability to hide cells based on cell tags

### PR DESCRIPTION
**Pull request by AartGoossens**
This PR documents the ability to hide code and output cells based on cell tags.
Related issue: https://github.com/voila-dashboards/voila/issues/171

### CODE REVIEW

**Comment by AartGoossens** [comment]
I also changed some headings to proper markdown headings so you can directly link to subsections in the README.

**Comment by jtpio** [comment]
Many thanks @AartGoossens for this.

I wonder if these instructions could also go to the documentation, under the customizing section? https://voila.readthedocs.io/en/latest/customize.html

**Comment by jtpio** [comment]
And this should also fix https://github.com/voila-dashboards/voila/issues/171.

**Comment by AartGoossens** [comment]
I updated the PR: moved the documentation to `docs/source/customize.rst` (which is indeed a much better place).

Let me know what else I can do before this can be merged.

File: docs/source/customize.rst [hunk 1]
```diff
@@ -232,3 +232,20 @@ Python this would be a call to `IPython.display.display`.
 Using `Jupytext <https://github.com/mwouts/jupytext>`_ is another way to support
 script files. After installing jupytext, voila will see script files as if they
 are notebooks, and requires no extra configuration.
+
+Hiding ouput and code cells based on cell tags
```
Comment (jtpio) [inline docs/source/customize.rst]:
```suggestion
Hiding output and code cells based on cell tags
```

File: docs/source/customize.rst [hunk 2]
```diff
@@ -232,3 +232,20 @@ Python this would be a call to `IPython.display.display`.
...
+==============================================
+
+Voila uses `nbconvert <https://github.com/jupyter/nbconvert>`_ under the hood to render the notebooks so we can benefit from some of it's advanced functionality to hide code and output cells based on cell tags.
```
Comment (jtpio) [inline docs/source/customize.rst]:
```suggestion
Voila uses `nbconvert <https://github.com/jupyter/nbconvert>`_ under the hood to render the notebooks so we can benefit from some of its advanced functionality to hide code and output cells based on cell tags.
```

File: docs/source/customize.rst [hunk 3]
```diff
@@ -232,3 +232,20 @@ Python this would be a call to `IPython.display.display`.
...
+
+To hide the cell output for every cell in your notebook that has been tagged (`how to tag <https://jupyter-notebook.readthedocs.io/en/stable/changelog.html#cell-tags>`_) with "hide" in Voila::
+    
+    voila --TagRemovePreprocessor.remove_all_outputs_tags='{"hide"}' your_notebook.ipynb
+
+To hide both the code cell and the output cell (if any) for every cell that has been tagged with "hide"::
+
+    voila --TagRemovePreprocessor.remove_cell_tags='{"hide"}' your_notebook.ipynb
+
+You can use any tag you want to use this functionality but be sure to use the same tag name in the Voila command.
```
Comment (jtpio) [inline docs/source/customize.rst]: Maybe a bit more concise?

```suggestion
You can use any tag you want but be sure to use the same tag name in the Voila command.
```

**Comment by jtpio** [comment]
Thanks again @AartGoossens, left a couple of inline comments (minor typos).

**Comment by AartGoossens** [comment]
@jtpio Thanks for taking a good look and noticing the typos. :)
I fixed them and pushed my changes.

**Comment by maartenbreddels** [comment]
good catch @SylvainCorlay :)

**Pull request merged**